### PR TITLE
fix: use import command instead of editing config files

### PR DESCRIPTION
**Pull request by richardmcclellan**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by medelman17** [comment]
@richardmcclellan, at first glance, this seems like a big change. Can you open/link an issue with context--ideally, a user story--so that we can better understand the implications, here? In other words, there seems like there might be a lot of ancillary issues that might need to be documented as a result of this change. For example, 'how to migrate your data from one AppSync api to another`.

**Comment by richardmcclellan** [comment]
> @richardmcclellan, at first glance, this seems like a big change. Can you open/link an issue with context--ideally, a user story--so that we can better understand the implications, here? In other words, there seems like there might be a lot of ancillary issues that might need to be documented as a result of this change. For example, 'how to migrate your data from one AppSync api to another`.

I've updated this PR to be a little bit less aggressive of a change :).    The `amplify import` command is only supported for `auth` and `storage`, so this PR now only updates those categories to indicate that using the `amplify import` command is another option, in addition to direct modification of the `amplifyconfiguration.json` file.

File: docs/lib/auth/fragments/existing-resources.md [hunk 1]
```diff
@@ -1,4 +1,14 @@
-Existing Amazon Cognito Identity Pool and User Pool can be used with the Amplify Libraries by referencing them in your `amplifyconfiguration.json` file.
+
+If you are using the Amplify CLI, an existing Amazon Cognito Identity Pool and User Pool can be used with the Amplify Libraries by running:
```
Comment (renebrandel) [inline docs/lib/auth/fragments/existing-resources.md:2]:
```suggestion
If you are using the Amplify CLI, an existing Amazon Cognito User Pool and Identity Pool can be used with the Amplify Libraries by running:
```

File: docs/lib/auth/fragments/existing-resources.md [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+
+```bash
+amplify import auth
+```
+
+Refer to [the related CLI documentation](~/cli/auth/import.md) for more details.
+
+
+If you are not using the Amplify CLI, a Cognito Identity Pool and User Pool can be used by referencing them in your `amplifyconfiguration.json` file.
```
Comment (renebrandel) [inline docs/lib/auth/fragments/existing-resources.md:11]:
```suggestion
If you are not using the Amplify CLI, a Cognito User Pool and Identity Pool can be used by referencing them in your `amplifyconfiguration.json` file.
```

File: docs/lib/auth/fragments/existing-resources.md [hunk 3]
```diff
@@ -1,4 +1,14 @@
-Existing Amazon Cognito Identity Pool and User Pool can be used with the Amplify Libraries by referencing them in your `amplifyconfiguration.json` file.
+
+If you are using the Amplify CLI, an existing Amazon Cognito Identity Pool and User Pool can be used with the Amplify Libraries by running:
+
+```bash
+amplify import auth
+```
+
+Refer to [the related CLI documentation](~/cli/auth/import.md) for more details.
```
Comment (renebrandel) [inline docs/lib/auth/fragments/existing-resources.md:8]: Use AWS writing style guide approach here. Example:
"For more information about <topic>, see [Page title](link)."

File: docs/lib/storage/fragments/existing-resources.md [hunk 4]
```diff
@@ -1,4 +1,12 @@
-An existing Amazon S3 bucket can be used with the Amplify Libraries by referencing it in your `amplifyconfiguration.json` file.
+If you are using the Amplify CLI, an existing Amazon S3 bucket can be used with the Amplify Libraries by running:
+
+```bash
+amplify import storage
+```
+
+Refer to [the related CLI documentation](~/cli/storage/import.md) for more details.
```
Comment (renebrandel) [inline docs/lib/storage/fragments/existing-resources.md:7]: Same comment as above.

File: docs/lib/auth/fragments/existing-resources.md (hunk 3 above)
Comment (richardmcclellan) [reply docs/lib/auth/fragments/existing-resources.md:8]:
```suggestion
For more details, see [the related CLI documentation](~/cli/auth/import.md).
```

File: docs/lib/auth/fragments/existing-resources.md [hunk 5]
```diff
@@ -1,4 +1,14 @@
-Existing Amazon Cognito Identity Pool and User Pool can be used with the Amplify Libraries by referencing them in your `amplifyconfiguration.json` file.
+
+If you are using the Amplify CLI, an existing Amazon Cognito User Pool and Identity Pool can be used with the Amplify Libraries by running:
+
+```bash
+amplify import auth
+```
+
+For more details, see [the related CLI documentation](~/cli/auth/import.md).
```
Comment (richardmcclellan) [inline docs/lib/auth/fragments/existing-resources.md:8]:
```suggestion
For more details, see how to [Use an existing Cognito User Pool and Identity Pool](~/cli/auth/import.md).
```

File: docs/lib/storage/fragments/existing-resources.md (hunk 4 above)
Comment (richardmcclellan) [reply docs/lib/storage/fragments/existing-resources.md:7]:
```suggestion
For more details, see how to [Use an existing S3 bucket or DynamoDB table](~/cli/storage/import.md).
```

**Review by renebrandel — APPROVED** [review]

**Review by medelman17 — APPROVED** [review]

**Pull request merged**